### ISSUE TITLE
Add .Get() to Tuple family as common alias for .Unpack() to match samber/mo

### ISSUE THREAD

**Issue by jcbhmr** [samber/lo#448]
basically this is the other end of https://github.com/samber/mo/issues/40 -- implement .Get() -> (T, U, V, ...) for the Tuple family of types instead of having samber/mo's Result and Option implement .Unpack()

```go
func (t *TupleN) Get() (T, U, V, ...) {
  return t.A, t.B, t.C, ...
}
```

why do I want this? so that I can use https://github.com/samber/mo Option Result and this package's Tuple interchangeably and just return an interface like this:

```go
type myGet[T any, U any] interface {
  Get() (T, U)
}

func MyFunc() <-chan myGet[int, error] {
  c := make(chan myGet[int, error])
  go func(){
    defer close(c)
    time.Sleep(200 * time.Milliseconds)
    // c <- mo.Option
    // c <- lo.Tuple2
    // c <- mo.Result
    // c <- whatever
  }()
  return c
}
```

```go
import "github.com/jcbhmr/thelib"

func main() {
  v, err := (<-thelib.MyFunc()).Get()
  if err != nil {
    log.Fatal(err)
  }
  log.Printf("The result is %d", v)
}
```

tldr: id like to see Unpack() or Get() be the one that is common among samber/lo and samber/mo
